### PR TITLE
Alert dialog overlay 38

### PR DESCRIPTION
**Pull request by roblevintennis**
This fixes https://github.com/morkro/vue-a11y-dialog/issues/38

### CODE REVIEW

File: src/alertdialog/AlertDialog.spec.ct.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
+import { mount } from '@cypress/vue'
+import AlertDialog from './AlertDialog.vue'
+
+/**
```
Comment (roblevintennis) [inline src/alertdialog/AlertDialog.spec.ct.js:4]: So one compromise of using a lower-level lib like `a11y-dialog` that evaluates data attributes on instantiation, is that we can't easily use reactivity e.g. to update the role between dialog and alertdialog and have those changes reflected dynamically. In reality, consumers are going to likely make this choice of role ahead of time so I don't see it as a big deal.

It does require what I've done here which is to create a new instance separately. I've placed it in its own `/alertdialog/` sub-directory to not draw attention away from the Demo.vue stuff which I believe we wanted to have the dual purpose of being a test fixture but also a demo.

**Comment by morkro** [comment]
Reviewing later today :)

File: src/alertdialog/AlertDialog.spec.ct.js (hunk 1 above)
Comment (morkro) [reply src/alertdialog/AlertDialog.spec.ct.js:4]: Not a huge fan of creating another duplicate demo+test file here. Can't we just have multiple dialogs in the demo file showcasing the different options? Alternatively, we don't necessarily need to have a separate demo file for `alertdialog`. I am happy to just remove it from the demo as it's well explained in the docs.

Comment (roblevintennis) [reply src/alertdialog/AlertDialog.spec.ct.js:4]: I'll just remove it when I get back to a computer later￼

Comment (roblevintennis) [reply src/alertdialog/AlertDialog.spec.ct.js]: @morkro Ok, that's been pushed — please re-review ;-)

**Comment by roblevintennis** [comment]
I think this is good to go  re...

> I am happy to just remove it from the demo as it's well explained in the docs.

That's been removed

**Comment by morkro** [comment]
Thanks @roblevintennis !

**Comment by roblevintennis** [comment]
Thank you!

**Pull request merged**